examples/iaas_and_iaasalpha/resourse_example.py

```diff
@@ -0,0 +1,187 @@
+alpha_client = AlphaDefaultApi(config)
+resource_client = ResourceDefaultApi(config)
+
+admin_vps_routes = ["147.204.0.0/16", "100.64.0.0/10", "8.8.8.8/32"]
+
+create_network_area_payload = CreateNetworkAreaPayload(
+    name="demo-network-area",
+    addressFamily=CreateAreaAddressFamily(
+        ipv4=CreateAreaIPv4(
+            networkRanges=[
+                NetworkRange(prefix="10.0.0.0/16"),
+            ],
+            transferNetwork="192.160.0.0/24",
+            default_nameservers=["9.9.9.9"],
+        )
+    ),
+)
+demo_network_area = client.create_network_area(organization_id, create_network_area_payload)
+
+
+project = resource_client.create_project(
+    CreateProjectPayload(
+        name="demo_network_area",
+        container_parent_id=organization_id,
+        labels={"networkArea": demo_network_area.area_id},
+        members=[Member(role="owner", subject="melvin.klein@stackit.cloud")],
+    )
+)
+project_id = project.project_id
+demo_network_area_area_id = demo_network_area.area_id
+
+default_network_creation_response = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="default",
+    ),
+)
+
+security_group1 = alpha_client.create_security_group(
+    project_id,
+    CreateSecurityGroupPayload(
+        name="My-security-group",
+        description="This is created via Python SDK",
+        labels={"key": "value"},
+    ),
+)
+security_group1_groupid = security_group1.id
+
+gateway_a_network = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="gateway_a_network",
+    ),
+)
+
+gateway_b_network = client.create_network(
+    project_id,
+    CreateNetworkPayload(
+        name="gateway_b_network",
+    ),
+)
+
+gateway_a_nic1 = alpha_client.create_nic(
+    project_id,
+    gateway_a_network.network_id,
+    CreateNICPayload(
+        allowed_addresses=[x for x in admin_vps_routes],
+        security_groups=[security_group1_groupid],
+    ),
+)
+
+gateway_a_public_ip = alpha_client.create_public_ip(
+    project_id, CreatePublicIPPayload(network_interface=gateway_a_nic1.id)
+)
+
+gateway_b_nic1 = alpha_client.create_nic(
+    project_id,
+    gateway_b_network.network_id,
+    CreateNICPayload(
+        allowed_addresses=[x for x in admin_vps_routes],
+        security_groups=[security_group1_groupid],
+    ),
+)
+
+gateway_b_public_ip = alpha_client.create_public_ip(
+    project_id, CreatePublicIPPayload(network_interface=gateway_b_nic1.id)
+)
+
+gateway_a = alpha_client.create_server(
+    project_id,
+    CreateServerPayload(
+        name="gateway-a",
+        boot_volume=BootVolume(
+            size=64,
+            source=BootVolumeSource(
+                id="db10714b-5d17-4a88-83ea-cb499530acc8",
+                type="image",
+            ),
+        ),
+        keypair_name="key",
+        labels={"key": "value"},
+        machine_type="t1.2",
+        availability_zone="eu01-1",
+    ),
+)
+time.sleep(180)  # wait for server to be created
```

Comment:
add a comment stating that is is a temporary sleep until we have waiter methods implemented